includes/pipeline_page/docs_schema.php

```diff
@@ -170,6 +171,16 @@ function print_param($is_group, $param_id, $param, $is_required = false) {
             $row_class .= ' param-docs-hidden collapse d-print-block';
         }
 
+        # Body
+        $param_body =
+            '<div id="' .
```

Comment:
Ugh, looks grim when split over a line. Might start using `sprintf` a bit more, now that we're using prettier..